lib/yard/handlers/ruby/visibility_handler.rb

```diff
@@ -13,10 +13,22 @@ class YARD::Handlers::Ruby::VisibilityHandler < YARD::Handlers::Ruby::Base
     case statement.type
     when :var_ref, :vcall
       self.visibility = ident.first.to_sym
-    when :fcall, :command
+    when :command
+      if RUBY_VERSION >= '3.' && is_attribute_method?(statement.parameters.first)
```

Comment:
Ruby version check seems unnecessary since (a) this will check the version of Ruby parsing the source file, not the version that the source file was intended for and (b) since (a) is true, as long as ripper parses correctly, we should assume the code was written for the latest Ruby (we don't know what the target Ruby the .rb file was written for).

```suggestion
      if is_attribute_method?(statement.parameters.first)
```